docs/content/latest/troubleshoot/cluster/failed_tablets.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Recover failed YB-TServer for failed tablets
+linkTitle: Recover failed YB-TServer for failed tablets
+description: Recover failed YB-TServer for failed tablets
+aliases:
+  - /troubleshoot/cluster/recover-tserver-failed-tablets/
+  - /latest/troubleshoot/cluster/recover-tserver-failed-tablets/
+menu:
+  latest:
+    parent: troubleshoot-cluster
+    weight: 830
+isTocNested: true
+showAsideToc: true
+---
+
+When a YB-TServer or YB-Master process or node has failed, YugabyteDB will automatically trigger a remote bootstrap for
```

Comment:
let's not mention master here, as remote bootstraps for masters are a bit more tricky